src/pivot-table/hooks/use-column-width.ts

```diff
@@ -306,6 +316,15 @@ export default function useColumnWidth(
 
   const showLastRightBorder = useMemo(() => totalWidth < rect.width, [totalWidth, rect.width]);
 
+  // Horizontal scrollbar height control based on columns (full) visibility
+  useEffect(() => {
+    const allLeftGridColumnsVisible = leftGridWidthInfo.leftGridWidth === leftGridWidthInfo.leftGridFullWidth;
+    const allDataGridColumnsVisible = rightGridWidth === rightGridFullWidth;
+
+    if (allLeftGridColumnsVisible && allDataGridColumnsVisible) horizontalScrollbarHeightSetter(true);
```

Comment:
curly braches?

Reply:
it felt unnecessary since it's just only one line, but will fix it

Reply:
I understand, but for consistency I think we should use it